### DISCUSSION TITLE
Loading issue of Quasar documentation pages

### DISCUSSION THREAD

**Discussion by Ammamon** [quasarframework/quasar#11253] [General - Components / Directives / etc] [answered]
Past few days, Quasar documentation pages very slow. Many a time, the interactive examples are not even loaded.  Can this be please looked into?

See below an example of a page loading that continues forever.

![image](https://user-images.githubusercontent.com/7417777/140762110-69c92d99-92fa-4cbd-b725-c7a2446dfc8c.png)

**Comment by MilosPaunovic** [comment, marked as answer]
Does anyone else have similar problems? I, for one, could access same as always.

@Ammamon Have you tried checking whether it is something related to you connectivity issues (or other local factors)?

**Reply by Ammamon** [reply]
It could be then the connection issue. Please ignore this post.
